crawl-ref/source/spl-cast.cc

```diff
@@ -2689,7 +2671,9 @@ static int _spell_power(spell_type spell, bool evoked)
     if (cap == 0)
         return -1;
     const int pow = evoked ? wand_power()
-                           : calc_spell_power(spell, true, false, false);
+                           : calc_spell_power(spell);
+    //TODO: currently applies cap twice to nonwand spells.
+    //Double check that we can safely put cap in wand_power
```

Comment:
Should be fine, yeah. `wand_power()` is only used in one other place, `zap_wand`, where it's immediately passed to `your_spells`. All wand spells (listed in `spl-book.cc`) have non-zero power caps, and the only one with <200 power cap is throw flame, which is capped at 50. But so's the corresponding zap in zap-data.h, so I really doubt it matters.